File: Java/Fibonacci.java
import java.util.Scanner;

public class Fibonacci{
  public void chamarFibonacci(){

    Scanner prompt = new Scanner(System.in);

    int primeiro = 0;
    int segundo = 1;
    String sequenciaDeNumeros = "0 1";

    System.out.println("Digite a quantidade de numeros que voce quer que apareca na sequencia: ");
    int entrada = prompt.nextInt();

    for (int i = 0; i < entrada - 2; i++) {
      int soma = primeiro + segundo;
      sequenciaDeNumeros += " " + soma;
      int aux = soma;
      primeiro = segundo;
      segundo = soma;

    }
    System.out.println(sequenciaDeNumeros);
    
  }
}

File: fibonacci.js
let primeiro = 0
let segundo = 1
let sequenciaDeNumeros = '0 1'

let entrada = parseInt(prompt('Digite a quantidade de numeros que voce quer que apareca na sequencia: '))

for (let x = 0; x < entrada - 2; x++) {
  let soma = primeiro + segundo
  sequenciaDeNumeros += " " + soma
  let aux = soma
  primeiro = segundo
  segundo = soma

}
document.write(sequenciaDeNumeros + "<br>")